### ISSUE TITLE
Outbound security rule for App Gateway to App Services Subnet missing

### ISSUE THREAD

**Issue by Kumudaya** [Azure-Samples/microsoft-foundry-baseline#7]
According to the NSG security rules mentioned in [Baseline highly available zone-redundant web application](https://learn.microsoft.com/en-us/azure/architecture/web-apps/app-service/architectures/baseline-zone-redundant), security rule for allowing outbound access to AppServiceSubnet is missing. Additionally, the naming convention used for outbound access from AppGateway (AppPlan.Out.Allow.AzureMonitor) to Azure Monitor seems to differentiate compared to other names used within the same NSG.

**Comment by Kumudaya** [comment]
According to the NSG security rules mentioned in [Baseline highly available zone-redundant web application](https://learn.microsoft.com/en-us/azure/architecture/web-apps/app-service/architectures/baseline-zone-redundant), security rule  for allowing outbound access to AppServiceSubnet is missing. Additionally, the naming convention used for outbound access from AppGateway (AppPlan.Out.Allow.AzureMonitor) to Azure Monitor seems to differentiate compared to other names used within the same NSG.

**Comment by ckittel** [comment]
Good catch, @Kumudaya, there is no reason why Application Gateway should need access to the integration subnet.  I'll update the Microsoft Learn article and remove that.

As for the naming on them, we can look into that here in our next update pass.

**Comment by johndowns** [comment]
Closing this now that the PR to the architecture-center-pr is merged.
